.github/workflows/quality.yml

```diff
@@ -31,78 +31,49 @@ jobs:
         run: |
           ./gradlew knit
 
-  # ktlint for all the modules
-  ktlint:
-    name: Kotlin Linter
+  # Check the project: ktlint, detekt, lint
+  lint:
+    name: Android Linter
     runs-on: ubuntu-latest
     # Allow all jobs on main and develop. Just one per PR.
     concurrency:
-      group: ${{ github.ref == 'refs/heads/main' && format('ktlint-main-{0}', github.sha) || github.ref == 'refs/heads/develop' && format('ktlint-develop-{0}', github.sha) || format('ktlint-{0}', github.ref) }}
+      group: ${{ github.ref == 'refs/heads/main' && format('lint-main-{0}', github.sha) || github.ref == 'refs/heads/develop' && format('lint-develop-{0}', github.sha) || format('lint-{0}', github.ref) }}
       cancel-in-progress: true
     steps:
       - uses: actions/checkout@v3
       - name: Run ktlint
         run: |
           ./gradlew ktlintCheck --continue
+      - name: Run detekt
+        if: always()
+        run: |
+          ./gradlew detekt $CI_GRADLE_ARG_PROPERTIES
+      - name: Run lint
+        # Not always, if ktlint or detekt fail, avoid running the long lint check.
+        run: |
+          ./gradlew lintGplayRelease lintFdroidRelease --stacktrace $CI_GRADLE_ARG_PROPERTIES
       - name: Upload reports
         if: always()
         uses: actions/upload-artifact@v3
         with:
-          name: ktlinting-report
+          name: linting-report
           path: |
             */build/reports/ktlint/ktlint*/ktlint*.txt
-      - name: Handle Results
+            */build/reports/detekt/detekt.html
+            */build/reports/*.*
+      - name: Prepare Danger
         if: always()
-        id: ktlint-results
```

Comment:
This is not necessary anymore, since Danger will post comments for each detected ktlint issue